js/utils.js

```diff
@@ -792,7 +792,10 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
             return;
           }
 
-          if (entryData[column] && file.name.indexOf(entryData[column]) !== -1) {
+          // remove file extension 
+          var fileName = file.name.match(/(.+?)(?:\.[^\.]*$|$)/)[1];
+
+          if (entryData[column] && (file.name === entryData[column] || fileName === entryData[column])) {
```

Comment:
![image](https://user-images.githubusercontent.com/290733/72544001-806ec800-387e-11ea-9527-9482057b1d55.png)

```suggestion
          if (entryData[column] && fileName === entryData[column]) {
```

Reply:
In that case, it would not fully work, because the user has the possibility to write entryData[column] with extension. 
For example, dog.jpeg , in that case file.name after regexp would be simply dog.

Reply:
We use regexp on file names, which always do have the extension. But user can enter both dog and dog.jpeg

Reply:
Ah yes, you're right 👍